qupath-extension-bioformats/src/main/java/qupath/lib/images/writers/ome/ConvertCommand.java

```diff
@@ -341,6 +353,10 @@ private static Range getRange(String rangeText, int maxRange, String rangeLabel)
 		}
 	}
 
+	static boolean isValidRange(Range range, int maxRange) {
```

Comment:
Why not `private`?

Reply:
It depends if any tests will be added - if so, I think it has to be package private?

Reply:
If not, then yes to private.

Reply:
This will be tested via the convert command, not via the `isValidRange()` function. `isValidRange()` is an implementation detail, while tests are only testing the public behavior of a class.

Also, tests should never make the main code less robust (by leaking implementation details here).